docs/INSTALL.md

```diff
@@ -21,7 +21,7 @@
 - [Macaroons](#macaroons)
 - [Network Reachability](#network-reachability)
 - [Simnet vs. Testnet Development](#simnet-vs-testnet-development)
-- [Creating an lnd.conf (Optional)](#creating-an-lndconf-optional)
+- [Creating a lnd.conf (Optional)](#creating-an-lndconf-optional)
```

Comment:
it was correct before

Reply:
This is outdated, check the latest commit. Did a search and looks like I got all of them.

Reply:
ah - it isnt outdated as it is still in the first commit. I see you un-did the changes in a second commit though - these should be squashed into the first commit so that they are never changed

Reply:
Squashed! :)